chia/cmds/farm_funcs.py

```diff
@@ -219,13 +220,18 @@ class PlotStats:
     if all_harvesters is not None:
         harvesters_local: dict = {}
         harvesters_remote: dict = {}
+        harvesters_hostnames: dict = {}
         for harvester in all_harvesters["harvesters"]:
             ip = harvester["connection"]["host"]
             if is_localhost(ip):
                 harvesters_local[harvester["connection"]["node_id"]] = harvester
             else:
                 if ip not in harvesters_remote:
                     harvesters_remote[ip] = {}
+                    try:
+                        harvesters_hostnames[ip] = socket.gethostbyaddr(ip)[0]
```

Comment:
This call will take a long time to fail - many seconds - on systems when the reverse DNS lookup fails. With several harvesters this could delay by over a minute. I don't think we should do this by default - but as an added option it would be ok.

Reply:
Hmm not sure if we really should make such a nice feature optional just because in (probably?) rare special cases with a many harvesters + DNS lookup needs to fail for all of them for whatever reason it might take longer. Why would such a lookup take "many seconds" if it fails? Im not very into DNS stuff so can you elaborate here and please correct me if i miss something.

Reply:
DNS by default can have rather long timeouts and it would worry me about making this call slower. DNS can be hard to predict in the wild. It's possible though I'm being overly cautious

Reply:
I tried to get some slow responses from that method but i wasn't able to trigger some long running issue but i see what you mean, you never know what happens out there in the wild :smile:  Still since this is only CLI which gets triggered by the user i would not disable it because it could fail and take some more time for whatever setups.

Maybe someone else has more thoughts to share here but i would vote to have this at least enabled by default if its optional so that if we see issues popping up related to this we can say "add blabla to your config file" or we don't add yet another option to the config, just see how things go and include it to the next maintenance release if we see issues coming.

Reply:
This was discussed today and there was some general agreement in not adding in any possible blocking calls here. A config.yaml option is a good way to set this to it can be optional @srivatsanhari